### PR TITLE
fix(model): normalize model names to support flexible provider/model format

### PR DESCRIPTION
**Pull request by leermao**
## Related Issue
https://team.arcblock.io/comment/discussions/9mLWxHgFSDLVEGo9-fE6aq297rz1mdhxc/#5de51833-499e-49e2-b80d-80c9a7d35b82

### Major Changes
- Add formatHubModel to handle various model name formats (provider:model, provider/model, model)
- Auto-prefix unprefixed models with 'aignehub/'
- Add comprehensive tests covering edge cases"

### Screenshots
image
image
image
image

### Checklist

- [x] This change requires documentation updates, and I have updated the relevant documentation. If the documentation has not been updated, please create a documentation update issue and link it here
- [x] The changes are already covered by tests, and I have adjusted the test coverage for the changed parts
- [x] The newly added code logic is also covered by tests
- [x] This change adds dependencies, and they are placed in dependencies and devDependencies
- [x] This change includes adding or updating npm dependencies, and it does not result in multiple versions of the same dependency [check the diff of pnpm-lock.yaml]

### CODE REVIEW

File: observability/ui/src/libs/index.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import Decimal from "decimal.js";
 import type { TraceData } from "../components/run/types.ts";
 
+const MAX_STRING_LENGTH = 100;
```
Comment (li-yechao) [inline observability/ui/src/libs/index.ts:4]: 不要把不同模块不想关的改动放到一个 pr，changelog 会混乱

File: observability/ui/src/components/json-view.tsx [hunk 2]
```diff
@@ -57,6 +57,7 @@ const JsonView = forwardRef<JsonViewRef, JsonViewProps>(
 
     const processedData = useMemo(() => {
       if (!truncateStrings) return data;
+
```
Comment (li-yechao) [inline observability/ui/src/components/json-view.tsx]: 同上

File: models/aigne-hub/src/index.ts [hunk 3]
```diff
@@ -153,7 +159,7 @@ export class AIGNEHubVideoModel extends VideoModel {
     let provider = process.env.BLOCKLET_AIGNE_API_PROVIDER;
 
     if (!provider && options.model) {
-      const parsed = parseModel(options.model);
+      const parsed = parseModel(formatHubModel(options.model));
```
Comment (li-yechao) [inline models/aigne-hub/src/index.ts]: 应该直接修改 parseModel 方法

File: models/aigne-hub/src/utils/model.ts [hunk 4]
```diff
@@ -271,7 +271,15 @@ export function findVideoModel(provider: string): {
 
 export const parseModel = (model: string) => {
   // replace first ':' with '/' to compatible with `provider:model-name` format
-  model = model.replace(/^([\w-]+)(:)/, "$1/");
+  model = formatHubModel(model.replace(/^([\w-]+)(:)/, "$1/"));
```
Comment (li-yechao) [inline models/aigne-hub/src/utils/model.ts]: 这里的逻辑应该是：
1. 尝试获取 provider name
2. 从 providers 中根据 provider name 查找 provider
3. 找到了就用 provider，找不到则使用 aignehub

现在的实现存在一个问题：如果一个 model name 本身包含 `/` 会导致其无法使用 aignehub

File: models/aigne-hub/src/utils/model.ts [hunk 5]
```diff
@@ -271,7 +271,25 @@ export function findVideoModel(provider: string): {
...
-  model = model.replace(/^([\w-]+)(:)/, "$1/");
-  const { provider, name } = model.match(/(?<provider>[^/]*)(\/(?<name>.*))?/)?.groups ?? {};
-  return { provider: provider?.replace(/-/g, ""), model: name };
+  model = model.replace(/^([\w-]+):/, "$1/");
+  let { provider, name } = model.match(/(?<provider>[^/]*)(\/(?<name>.*))?/)?.groups ?? {};
+
+  const all = availableModels();
+  provider = provider?.replace(/-/g, "");
+
+  const match = provider
```
Comment (li-yechao) [inline models/aigne-hub/src/utils/model.ts]: 直接调用 `findModel` 即可？这里逻辑和 findModel 重复了

**Review by li-yechao — APPROVED** [review]
